put_get_test.go

```diff
@@ -236,7 +236,7 @@ func TestPutLocalFile(t *testing.T) {
 	})
 }
 
-func TestPutWithAutoCompressFalse(t *testing.T) {
+func TestPutGetWithAutoCompressFalse(t *testing.T) {
 	if runningOnGithubAction() && !runningOnAWS() {
```

Comment:
Is this still valid? Maybe we can run for all clouds?

Reply:
Works locally for me so I have removed the condition and will check later if the test passes on github.

Reply:
The tests passed on all clouds.